horoscope/horoscope.py

```diff
@@ -274,15 +245,18 @@ async def _font(self, ctx, url: str = None):
         await ctx.send("Font has been saved")
 
     def fortune_process(self, fortune):
-        img = Image.open(str(bundled_data_path(self) / "cookie.png"))
-        draw = ImageDraw.Draw(img)
-        font = ImageFont.truetype(str(bundled_data_path(self) / "FortuneCookieNF.ttf"), 15)
-        line = fortune.split()
-        sep = " "
-        line1 = sep.join(line[:5])
-        line2 = sep.join(line[5:10])
-        line3 = sep.join(line[10:])
-        draw.text((134, 165), line1, (0, 0, 0), font=font, align="center")
-        draw.text((134, 180), line2, (0, 0, 0), font=font, align="center")
-        draw.text((134, 195), line3, (0, 0, 0), font=font, align="center")
-        img.save(str(bundled_data_path(self) / "cookie-edit.png"))
+        with Image.open(str(bundled_data_path(self) / "cookie.png")) as img:
+            draw = ImageDraw.Draw(img)
+            font = ImageFont.truetype(str(bundled_data_path(self) / "FortuneCookieNF.ttf"), 15)
+            line = fortune.split()
+            sep = " "
+            line1 = sep.join(line[:5])
+            line2 = sep.join(line[5:10])
+            line3 = sep.join(line[10:])
+            draw.text((134, 165), line1, (0, 0, 0), font=font, align="center")
+            draw.text((134, 180), line2, (0, 0, 0), font=font, align="center")
+            draw.text((134, 195), line3, (0, 0, 0), font=font, align="center")
+            img_buffer = BytesIO()
+            img.save(img_buffer, format='PNG')
+            img_buffer.seek(0)
+            return img_buffer
```

Comment:
so correct me if I'm wrong, now there is no persistent storing of the edited cookie, but it now exists as an image buffer which is then emptied into the discord.File uploader, yeah?